src/Microsoft.ML.Data/Evaluators/Metrics/BinaryClassificationMetrics.cs

```diff
@@ -19,7 +19,7 @@ public class BinaryClassificationMetrics
         /// a randomly chosen positive instance higher than a randomly chosen negative one
         /// (assuming 'positive' ranks higher than 'negative').
         /// </remarks>
-        public double Auc { get; }
+        public double AreaUnderTheRocCurve { get; }
```

Comment:
>AreaUnderTheRocCurve [](start = 22, length = 20)

I'd vote for `AreaUnderRocCurve` as be to parallel with `AreaUnderPrecisionRecallCurve`. If we do keep the `The` here, then we should add it to the AUPRC too.